lib/slim.js

```diff
@@ -1,59 +1,25 @@
 const isWsl = require('is-wsl');
+const glob = require('glob-all');
+const fse = require('fs-extra');
 
-/**
- * Get commands to slim the installed requirements
- * only for non-windows platforms:
- * works for docker builds and when run on UNIX platforms (wsl included)
- * @param {Object} options
- * @param {string} folderPath
- * @return {Array.<String>}
- */
-function getSlimPackageCommands(options, folderPath) {
-  let stripCmd = [];
+const getStripCommand = (options, folderPath) =>
+  process.platform !== 'win32' || isWsl || options.dockerizePip
+    ? ` && find ${folderPath} -name "*.so" -exec strip {} +`
```

Comment:
This again did not work on Win10 when I tried 😅  
> Serverless: Docker Image: lambci/lambda:build-python3.6
Error --------------------------------------------------
... 
find: ‘strip’: No such file or directory

But now having changed it back to a semicolon with a different escape seems to work and supposedly should also be fine for *nix platforms...  Could you try `` `  && find ${folderPath} -name "*.so" -exec strip {} ';' ` `` _(yes, single quotes :D)_
Sorry for making it so absurd to try different ways every time 😅 

But it will call the `strip` command _per each_ found `.so` file, whereas `+` should've processed as much files at once as possible (which might actually still be one per call)

Reply:
That's so odd. I'm fine with it being a `strip` call per `.so`. Just switched to `';'`. Lets see if CI/*nix is happy. Windows should be happy per your experiments.